modules/mirror-registry-introduction.adoc

```diff
@@ -0,0 +1,25 @@
+== Mirror registry for {product-title} introduction
+
+For disconnected deployments of {product-title}, a container registry is required to carry out the installation of the clusters. If users want to run a production-grade registry service on such a cluster, a separate registry deployment is required to install the first cluster. The mirror registry addresses this need and is included in every OpenShift subscription. It is available for download on the link:https://catalog.redhat.com/software/containers/openshift/mirror-registry-rhel8[mirror registry for {product-title}] page.
+
+The mirror registry allows users to install a small-scale version of Red Hat Quay and its required components using the `mirror-registry` command line interface (CLI) tool. The mirror registry is deployed automatically with pre-configured local storage and a local database. It also includes auto-generated user credentials and access permissions with a single set of inputs and no additional configuration choices to get started.
+
+The mirror registry provides a pre-determined network configuration and reports deployed component credentials and access URLs upon success. A limited set of optional configuration inputs like fully qualified domain name (FQDN) services, superuser name and password, and custom TLS certificates are also provided. This provides users with a container registry so that they can easily create an offline mirror of all {product-title} release content when running {product-title} in restricted network environments.
+
+The mirror registry is limited to hosting images that are required to install a disconnected {product-title} cluster, such as Release images or Red Hat Operator images. It uses local storage on your {op-system-base-full} machine, and storage supported by {op-system-base} is supported by the mirror registry. Content built by customers should not be hosted by the mirror registry.
+
+Unlike Red Hat Quay, the mirror registry is not a highly-available registry and only local filesystem storage is supported. Although it is technically possible, using the mirror registry with more than one cluster is discouraged, because multiple clusters can create a single point of failure when updating your cluster fleet. It is advised to leverage the mirror registry to install a cluster that can host a production-grade, highly-available registry such as Red Hat Quay, which can serve {product-title} content to other clusters.
+
+Use of the mirror registry is optional if another container registry is already available in the install environment.
+
+The mirror registry is comprised of the following images, which can be found in the link:https://catalog.redhat.com/software/containers/explore[Red Hat Ecosystem Catalog]:
+
+* Quay: registry.redhat.io/quay/quay-rhel8:v3.6.1
+* Redis: registry.redhat.io/rhel8/redis-6:1-25
+* Postgres: registry.redhat.io/rhel8/postgresql-10:1-161
+* Mirror registry: quay.io/quay/mirror-registry-ee:latest
```

Comment:
Wondering whether we need to link these images. They are all part of the payload. Do we need this? We might confuse users into having to download these.